doc/examples/plot_multiblock_local_binary_pattern.py

```diff
@@ -0,0 +1,82 @@
+"""
+===========================================================
+Multi-Block Local Binary Pattern for texture classification
+===========================================================
+
+In this example, we will see how to compute the multi-block
```

Comment:
This example shows how to compute multi-block local binary pattern features as well as how to visualize them.